Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,21 +1,18 @@
 import React from 'react';
 import './App.css';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';
+import NumberButton from './components/ButtonComponents/NumberButton';
+import ActionButton from './components/ButtonComponents/ActionButton';
 
 const App = () => {
   return (
     <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+      <CalculatorDisplay />
+      <ActionButton text={"clear"} text={"/"} />
+      <NumberButton text="7" text="8" text="9" text="x" />
+      <NumberButton text="4" text="5" text="6" text="x" />
+      <NumberButton text="1" text="2" text="3" text="x" />
```

Comment:
The idea behind the `<ActionButton />` and `<NumberButton />` components is to represent a single button (as opposed to a row of buttons). You can then arrange these individual buttons inside `div` elements representing rows. Make sure to pass props too for button classes and text too. For example:
```
<div className="calculator-row">
    <ActionButton class="action-button" text="clear" />
    <NumberButton class="operator-button" text="÷" />
</div>
<div className="calculator-row">
    <NumberButton  class="number-button" text="7" />
    <NumberButton  class="number-button" text="8" />
    <NumberButton  class="number-button" text="9" />
    <NumberButton class="operator-button" text="×" />
</div>
```

For controlling the sizes of the buttons, you can use CSS to fit them inside a row (1 `<ActionButton />` component has a width of 3 `<NumberButton />` components).